pegasus/sites.v3/code.org/public/css/interactive_map.css

```diff
@@ -0,0 +1,28 @@
+#map {
+  width: 700px;
+  min-height: 425px;
+  float: left;
+  margin-right: -100px;
+}
+
+#infobox {
+  text-align: left;
+  float: right;
+}
+
+.info {
+  font-size: 18px;
+}
+
+#state-dropdown {
+  width: 70%;
+  height: 34px;
+  background-color: rgb(226,228,227); color: rgb(89,89,89);
```

Comment:
I also like only using RGB when we need RGBA, and that way we know immediately that we're changing opacity.